app/assets/stylesheets/application.bootstrap.scss

```diff
@@ -11,11 +11,6 @@ $container-max-widths: (
 @import "bootstrap-icons/font/bootstrap-icons";
 @import "dropzone";
 
-// Stop the page from jumping when switching, e.g., to a tab containing a scrollbar
-html {
-  margin-left: calc(100vw - 100%);
-}
```

Comment:
This is what caused the Bootstrap tooltip to be mis-aligned. Moved this CSS rule from `html` to `body`.